data/fields/contact/facebook.json

```diff
@@ -0,0 +1,6 @@
+{
+    "key": "contact:facebook",
+    "type": "url",
+    "label": "Facebook profile URL",
```

Comment:
As the tag allows to contain both facebook usernames or full URLs, we should probably use a more fitting name.

PS: Field labels should be in title case

```suggestion
    "label": "Facebook Username or URL",
```

Reply:
I think encouraging the username will result in a lot of bad data, people adding the profile title/label, not any kind of unique identifier. But we can be review it in the future anyway.